client/src/components/ui/PageUnavailable.svelte

```diff
@@ -18,5 +21,6 @@
 <style>
     div {
         text-align: center;
+        margin-top: var(--spacing-large);
```

Comment:
Maybe we can do without this since the parent `<section>` already has sufficient margin.
```suggestion
```